README.md

```diff
@@ -593,7 +593,8 @@ you proceed to [step 4](#verify-itts-client-version).
 
 One of the key components remote attestation is based on is the runtime measurement values (stored in RTMRs
 registers for each TD by the TDX module). These RTMR values are computed from the digests of the entries of
-the TD boot event log.
+the TD boot event log. During the system's boot, each event's digest (SHA384) is extended (appended) to the
```

Comment:
If you're using "appended" to clarify the meaning of "extended", maybe we should just go with "appended" unless "extended" is baked into the language of this component

Reply:
Done 57bef18762575d7cfbabc876ec221d46df674c1e